src/main/kotlin/uk/gov/justice/digital/hmpps/pollpush/config/ResourceServerConfiguration.kt

```diff
@@ -47,7 +47,7 @@ open class ResourceServerConfiguration : ResourceServerConfigurerAdapter() {
         .and().csrf().disable()
         .authorizeRequests()
         .antMatchers("/webjars/**", "/favicon.ico", "/csrf",
-            "/health", "/info", "/health/ping").permitAll()
+            "/health/**", "/info").permitAll()
```

Comment:
not sure why we've got spring security for this service as it doesn't expose any endpoints anyway :shrug: